app/src/main/java/org/simple/clinic/recentpatientsview/RecentPatientItem.kt

```diff
@@ -43,22 +45,25 @@ data class RecentPatientItem(
       uiEvents.onNext(RecentPatientItemClicked(patientUuid = uuid))
     }
 
-    viewHolder.render(name, age, gender, updatedAt, dateFormatter, clock)
+    viewHolder.render(name, age, gender, updatedAt, dateFormatter, clock, isNewRegistration)
   }
 
   class RecentPatientViewHolder(rootView: View) : ViewHolder(rootView) {
     private val nameAgeTextView by bindView<TextView>(R.id.recentpatient_item_title)
     private val lastSeenTextView by bindView<TextView>(R.id.recentpatient_item_last_seen)
     private val genderImageView by bindView<ImageView>(R.id.recentpatient_item_gender)
+    private val newRegistrationTextView by bindView<TextView>(R.id.recentpatient_new_registration)
```

Comment:
While we are here, should we switch to synthetic view accessors?

Reply:
Yeah let's do it

Reply:
I am going to open a different PR for this change since I will be converting the groupie usage with our regular adapter usage. Some of the changes may affect `RecentPatientsScreen` and it's items so I want to avoid doing that in this PR.